src/pages/bptl.js

```diff
@@ -30,7 +30,7 @@ export const bptlScreen = async (auth, route) => {
     unAuthorizedUser();
   return;
   }
-  bptlScreenTemplate(name, response.data, auth, route);
+  bptlScreenTemplate(name || response.data.email, response.data, auth, route);
```

Comment:
The email property and value is being stored in the object from your changes to connectFaas. I referenced the PR that was merged earlier [connectFaas email added](https://github.com/episphere/connectFaas/pull/160/files)